### PR TITLE
Update tower-clean-uninstall-pipeline

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

**Comment by tremes** [comment]
https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/tower-clean-uninstall-tremes/6/

File: jobs/integr8ly/ocp3/tower/tower-clean-uninstall.yaml [hunk 1]
```diff
@@ -72,17 +72,20 @@
...
                             // This returns an external IP of a master node of a newly created cluster
                             def masterIP = sh(
                                 returnStdout: true,
                                 script: """
-                                    sudo oc login https://${clusterName}.${clusterDomainName} -u ${clusterAdminCredentials.clusterAdminUsername} -p ${clusterAdminCredentials.clusterAdminPassword} > /dev/null
-                                    sudo oc describe node `oc get nodes | grep master | head -n 1 | awk '{print \$1}'` | grep ExternalIP | awk '{print \$2}'
+                                    oc describe node `oc get nodes | grep master | head -n 1 | awk '{print \$1}'` | grep ExternalIP | awk '{print \$2}'
                                 """).trim()
-
+
```
Comment (pawelpaszki) [inline jobs/integr8ly/ocp3/tower/tower-clean-uninstall.yaml]: after removing the whitespace LGTM

Comment (tremes) [reply jobs/integr8ly/ocp3/tower/tower-clean-uninstall.yaml]: OK :)

Comment (tremes) [reply jobs/integr8ly/ocp3/tower/tower-clean-uninstall.yaml]: obviously I can't do it :D

Comment (pawelpaszki) [reply jobs/integr8ly/ocp3/tower/tower-clean-uninstall.yaml]: not a blocker for merging then :)

Comment (tremes) [reply jobs/integr8ly/ocp3/tower/tower-clean-uninstall.yaml]: Done. God bless vim! :)

**Pull request merged**